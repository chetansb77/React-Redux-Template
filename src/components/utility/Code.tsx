import React from "react";

interface CodeProps {
  codeSnippet: string;
}

function Code({ codeSnippet }: CodeProps): JSX.Element {
  return (
    <div className="code-snippet">
      {codeSnippet}
    </div>
  );
}

export default Code;
