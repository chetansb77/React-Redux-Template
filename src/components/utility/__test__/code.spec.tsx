import React from "react";
import renderer from "react-test-renderer";
import Code from "../Code";

it("error component renders correctly", () => {
  const tree = renderer.create(<Code codeSnippet="Error traceback" />).toJSON();
  expect(tree).toMatchSnapshot();
});
