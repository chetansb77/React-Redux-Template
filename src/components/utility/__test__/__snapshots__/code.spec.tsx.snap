// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`error component renders correctly 1`] = `
<div
  className="code-snippet"
>
  Error traceback
</div>
`;
